internal/rootcoord/dml_channels.go

```diff
@@ -155,7 +167,10 @@ func newDmlChannels(ctx context.Context, factory msgstream.Factory, chanNamePref
 		name := genChannelName(d.namePrefix, i)
 		ms, err := factory.NewMsgStream(ctx)
```

Comment:
Check topic whether has data or not,  we'd better add a constraint that specified topics should never be used.